internal/uvm/create.go

```diff
@@ -69,7 +69,11 @@ func newDefaultOptions(id, owner string) *Options {
 	}
 
 	if opts.ID == "" {
-		opts.ID = guid.New().String()
+		g, err := guid.NewV4()
```

Comment:
Perhaps we can move this GUID generation to occur in CreateLCOW/CreateWCOW if ID is ""? Then we can avoid the panic here.

Reply:
I don't have strong feelings on this, panic is the behavior we had before, though it's probably better if we don't bring down the process in this case.